server/index.js

```diff
@@ -20,6 +22,6 @@ app.get('/api/v1', (req, res) => {
 });
 
 
-app.listen(process.env.PORT, () => console.log(`Teamwork app listening on port ${process.env.PORT}!`));
+app.listen(port, () => console.log(`Teamwork app listening on port ${port}!`));
```

Comment:
we do not merge `development tools` to `production code`

Reply:
Thanks